features/spring/spring43/pom.xml

```diff
@@ -161,18 +159,6 @@
       <version>${portlet-api.version}</version>
     </dependency>
 
-    <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>httpcore-osgi</artifactId>
-      <version>${httpcore-osgi.version}</version>
```

Comment:
Do not remove these dependencies. They are being declared here for traceability reasons for the spring43 feature creation. Just remove the version declaration and let the dependency be managed by the root POM.

Reply:
Reverted.

Reply:
Dependencies declared & versions removed.